### PR TITLE
OSDOCS3047: Dev Preview: Cgroups v2 phase 2

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-3047

New section added to [Post Install -> Machine configuration tasks](https://deploy-preview-39619--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html#nodes-nodes-cgroups-2_post-install-machine-configuration-tasks).
Same section added to [Nodes -> Working with nodes](https://deploy-preview-39619--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#nodes-nodes-cgroups-2_nodes-nodes-working). 
New [Optional parameter added to Installation configuration parameters](https://deploy-preview-39619--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-configuration-parameters-optional_installing-aws-customizations).

### CODE REVIEW

**Comment by mburke5678** [comment]
@harche @giuseppe PTAL

**Comment by harche** [comment]
cc @rphillips

**Review by giuseppe — APPROVED** [review]
LGTM

**Comment by mburke5678** [comment]
@sunilcio Please take a look. Thank you in advance!

File: modules/nodes-nodes-cgroups-2.adoc [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="nodes-nodes-cgroups-2_{context}"]
+= Enabling cgroups version 2
+
+You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) on specific nodes in your cluster by using a machine config. The {product-title} process for enabling cgroups v2 disables all cgroups version 1 controllers and hierarchies.
+
+[NOTE]
+====
+The {product-title} cgroups version 2 feature is in Developer Preview and is not supported by Red Hat at this time.
```
Comment (kmccarron-rh) [inline modules/nodes-nodes-cgroups-2.adoc:13]: Is this still a Developer Preview?  I am told that we don't doc Developer Previews for a couple of reasons which include support concerns.

**Review by kmccarron-rh — COMMENTED** [review]
Hi @mburke5678 I reached out to several people and received the same guidance regarding developer previews. Please let me know if you have any questions.

File: modules/nodes-nodes-cgroups-2.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/nodes-nodes-cgroups-2.adoc:13]: I thought we didn't. But there are 14 files that reference Developer Preview features. We address Developer Preview features [here](https://docs.openshift.com/container-platform/4.9/rest_api/understanding-compatibility-guidelines.html#api-compatibility-common-terminology-dev-preview_compatibility-guidelines).

File: modules/nodes-nodes-cgroups-2.adoc [hunk 2]
```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-working.adoc
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="nodes-nodes-cgroups-2_{context}"]
+= Enabling cgroups version 2
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/nodes-nodes-cgroups-2.adoc:7]: Good idea. I am assuming that anyone who is working with these this is fairly expert. it is a good change.

File: modules/nodes-nodes-cgroups-2.adoc (hunk 1 above)
Comment (kmccarron-rh) [reply modules/nodes-nodes-cgroups-2.adoc:13]: It might be best to discuss with your CS.

File: modules/nodes-nodes-cgroups-2.adoc [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) on specific nodes in your cluster by using a machine config. The {product-title} process for enabling cgroups v2 disables all cgroups version 1 controllers and hierarchies.
+
+[NOTE]
```
Comment (kmccarron-rh) [inline modules/nodes-nodes-cgroups-2.adoc:11]: Should this be an IMPORTANT? 
Similar to this:
https://docs.openshift.com/container-platform/4.9/scalability_and_performance/ztp-deploying-disconnected.html#configuring_bios_for_du_hosts_ztp-deploying-disconnected

File: modules/nodes-nodes-cgroups-2.adoc (hunk 1 above)
Comment (nermina-redhat) [reply modules/nodes-nodes-cgroups-2.adoc:13]: @kmccarron-rh we did when I acked this with Michael. Please proceed.

File: nodes/nodes/nodes-nodes-working.adoc [hunk 4]
```diff
@@ -33,6 +33,8 @@ include::modules/nodes-nodes-kernel-arguments.adoc[leveloffset=+1]
 ifdef::openshift-webscale[]
 include::modules/nodes-nodes-rtkernel-arguments.adoc[leveloffset=+1]
 endif::openshift-webscale[]
+include::modules/nodes-nodes-cgroups-2.adoc[leveloffset=+1]
+
```
Comment (kmccarron-rh) [inline nodes/nodes/nodes-nodes-working.adoc:37]: I'm not sure if this is a concern, but I did notice that this causes multiple Additional Resource sections stacked together: https://deploy-preview-39619--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#additional-resources

**Review by kmccarron-rh — COMMENTED** [review]
A couple of additional comments.

File: nodes/nodes/nodes-nodes-working.adoc (hunk 4 above)
Comment (sagidlow) [reply nodes/nodes/nodes-nodes-working.adoc:37]: I agree, this looks a bit odd. Is there a way to get around this?

Comment (kmccarron-rh) [reply nodes/nodes/nodes-nodes-working.adoc]: Well done! 
lgtm!

**Pull request merged**